snippets/facets.liquid

```diff
@@ -273,75 +273,11 @@
                             </li>
                           {%- endfor -%}
                         </ul>
-                        {% comment %} No show more for no JS {% endcomment %}
-                        <ul
-                          class="{% if filter_type != 'vertical' %} facets__list{% endif %} no-js-list list-unstyled no-js"
```

Comment:
This section was hidden unless in a no-js context, controlled by this code in base.css:

```css
.no-js:not(html) {
  display: none !important;
}

html.no-js .no-js:not(html) {
  display: block !important;
}
```